onnxruntime/core/providers/cpu/math/element_wise_ops.h

```diff
@@ -365,7 +365,7 @@ struct BroadcastIterator {
   }
 
   void StartBroadcasting() {
-    deltas_.push_back(-count_);
+    deltas_.push_back(0 - count_);
```

Comment:
>0 - count_ [](start = 22, length = 10)

Count is size_t. This results in max value for the delta, Can you rewrite this as std::numeric_limits<size_t>::max() - count_ or something to that effect?

Reply:
@RyanUnderhill What do you think?

Reply:
I think that will cause a bug since it wants to be a negative number, not a huge unsigned number. Does `0LL - count_` make it happy?